packages/fast-components/src/switch/switch.template.ts

```diff
@@ -0,0 +1,43 @@
+import { html, when } from "@microsoft/fast-element";
+import { Switch } from "./switch";
+import { bool } from "../utilities";
+
+export const SwitchTemplate = html<Switch>`
+    <template
+        role="switch"
+        $aria-checked="${x => bool(x.checked)}"
+        $aria-required="${x => bool(x.required)}"
+        $aria-disabled="${x => bool(x.disabled)}"
+        $aria-readonly="${x => bool(x.readOnly)}"
+        $tabindex="${x => (bool(x.disabled) ? null : 0)}"
+        @keypress="${(x, c) => x.keypressHandler(c.event as KeyboardEvent)}"
+        @click="${(x, c) => x.clickHandler(c.event as MouseEvent)}"
+    >
+        ${when(
+            x => x.childNodes.length,
```

Comment:
We'll leave this as is for now, but change it once we get some new features. Just note that this will only work for initial render but if the children change later, the label won't be added/removed. Nothing to change for now.